recipes/gnali/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "gNALI" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "649ec1d872442da520ed5ca670a1fe64e3307fc695cce75ba2cdaa8b653af6a3" %}
```

Comment:
For the next version bump, I suggest to remove this variable and write the checksum directly below. It’s preferable to use Jinja2 variables only when they’re actually used more than once in the recipe.